components/TheNavBar.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <nav class="bg-white w-full shadow-md fixed flex items-center z-50">
+    <div class="container mx-auto px-4 flex justify-between">
+      <ul id="menu_list list-none" v-bind:class="{ active: toggle == true }">
+        <li><a class="text-lg font-bold no-underline" href="#about">Despre Alpha</a></li>
+        <li><a class="text-lg font-bold no-underline" href="#testimonials">Testimoniale</a></li>
+        <li><a class="text-lg font-bold no-underline" href="#register">Încriere</a></li>
+      </ul>
+      <button class="toggle" @click="toggle = !toggle">
+        <span v-bind:class="{ 'translate-y-1/2 rotate-45': toggle == true }" class="nav_btn_activ absolute bg-black w-9 h-1 rounded-sm transform translate-y-1.5 top-1/2"></span>
+        <span v-bind:class="{ 'translate-y-1/2 -rotate-45': toggle == true }" class="nav_btn_activ absolute bg-black w-9 h-1 rounded-sm transform -translate-y-1.5 top-1/2"></span>
+      </button>
+      <a href="#top-section"><img class="logo" src="../static/logo.svg" alt=""></a>
+    </div>
+  </nav>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      toggle: false
+    }
+  },
+  methods: {
+    handleScroll (event) {
+      if (window.scrollY > 0) {
+        document.querySelector('nav').classList.add('nav-on-scroll')
+        document.querySelector('.logo').classList.add('logo-on-scroll')
+      } else {
+        document.querySelector('nav').classList.remove('nav-on-scroll')
+        document.querySelector('.logo').classList.remove('logo-on-scroll')
+      }
+    }
+  },
+  beforeMount () {
+    window.addEventListener('scroll', this.handleScroll)
+  },
+  beforeDestroy () {
+    window.removeEventListener('scroll', this.handleScroll)
+  }
+}
+</script>
+
+<style>
+
+html {
+  scroll-behavior: smooth;
+}
+
+nav {
+  height: 80px;
+  transition: height 0.2s ease-in-out;
+}
+
+.nav-on-scroll {
+  height: 50px;
+}
+
+nav ul {
+  position: absolute;
+  display: block;
+  text-align: center;
+  background: #fff;
+  bottom: 0;
+  left: -100%;
+  width: 100%;
```

Comment:
`block absolute bottom-0 text-center bg-white -left-full w-full`

I bet you will even find what you want for transform and transition but it's a shame for me to do all the research for you. after all, this is your chance to learn how to skim documentation, not how to **skip** documentation :)